content/spin/see-what-people-have-built-with-spin.md

```diff
@@ -23,14 +31,24 @@ The 'Building a Social App with Spin' series covers the process and decision-mak
 * [Part 3.5: Go Postgres Usage](https://www.fermyon.com/blog/building-a-social-app-with-spin-3-5)
 * [Part 4: Key-Value Storage and Fermyon Cloud](https://www.fermyon.com/blog/building-a-social-app-with-spin-4)
 
+## Static Content Server
+
 [Serving Static Content via WebAssembly](https://www.fermyon.com/blog/serving-static-content-via-webassembly)
 
+## Bots
+
 [Bots With Spin and Fermyon Cloud](https://www.fermyon.com/blog/bots-with-spin-and-fermyon-cloud)
 
+## URL Shortner and QR Code Generator
```

Comment:
```suggestion
## URL Shortener and QR Code Generator
```

Reply:
Thanks, also updated the ToC. Nice catch!